src/relayclient/RelaySelectionManager.ts

```diff
@@ -74,6 +74,7 @@ export default class RelaySelectionManager {
             relayInfo: events[0]
           }
         } else {
+          // TODO: do not throw! The preffered relay may be removed since.
```

Comment:
I don't understand the sentence...

Reply:
If you define a preferred relay in a client but relay is removed we want to handle this situation gracefully. Now this will throw (not sure what happens next)

Reply:
GSN-38

Reply:
If you (dapp developer) defined a preferred relay, then it most likely means that this is your relay. If you remove it, then you should also update the client's code.